versions.yml

```diff
@@ -4,6 +4,20 @@ logstash: 8.0.0
 logstash-core: 8.0.0
 logstash-core-plugin-api: 2.1.16
 
+bundled_jdk:
+  linux:
+      vendor: "adoptopenjdk"
+      revision: 14
+      build: 36
+  mac:
+      vendor: "adoptopenjdk"
+      revision: 14
+      build: 36
+  windows:
+      vendor: "zulu"
+      revision: 11.39.15
```

Comment:
If we do choose to bundle Zulu instead of adopt open jdk we would need to keep the major versions the same

Reply:
We decided to use AdoptOpenJDK also for windows